src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
it's allowed to use only box-sizing: border-box; inside * selector. If you need to reset margin and padding for other tags just use other selectors

Example:
```
body {
  margin: 0;
}
```